pootle/static/js/shared/components/AboutDialog.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) Zing contributors.
+ *
+ * This file is a part of the Zing project. It is distributed under the GPL3
+ * or later license. See the LICENSE file for a copy of the license and the
+ * AUTHORS file for copyright and authorship information.
+ */
+
+import React from 'react';
+import { showModal } from './Modal';
+
+import './AboutDialog.css';
+
+export function showAboutDialog() {
+  showModal({
+    title: 'About this translation server...',
+    children: <AboutDialogContent />,
+    className: 'AboutDialog',
+  });
+}
+
+const AboutDialogContent = React.createClass({
+
+  render() {
+    return (
+      <div>
+        <div className="sideColumn">
+          <img src="/static/images/logo.svg" />
```

Comment:
This won't work in production environments. Use `s()` to wrap the path to the image, relative to the static directory.